datacite-draft-doi-handler/src/main/java/no/unit/nva/datacite/handlers/DraftDoiHandler.java

```diff
@@ -91,8 +91,8 @@ private static DoiClient defaultDoiClient() {
         return DoiClientFactory.getClient(configFactory, connectionFactory);
     }
 
-    private boolean doiRequestIsApproved(Publication publication) {
-        return publication.getDoiRequest().getStatus().equals(DoiRequestStatus.APPROVED);
+    private boolean doiIsRequested(Publication publication) {
+        return publication.getDoiRequest().getStatus().equals(DoiRequestStatus.REQUESTED);
```

Comment:
I like to turn these around with the constant being first.

DoiRequestStatus.REQUESTED.equals(publication.getDoiRequest().getStatus());